companies/holvi.json

```diff
@@ -0,0 +1,27 @@
+{
+    "slug": "holvi",
+    "relevant-countries": [
+        "at",
+        "de",
+        "fi"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Holvi Payment Services Oy",
+    "address": "Hämeentie 11\n00530 Helsinki\nFinland",
+    "phone": "+358 75 325 2935",
+    "fax": "+358 92 319 4337",
```

Comment:
Can you list a source for the phone and fax number? I didn't find them in the privacy policy.

Reply:
At the end of the privacy policy, there is an option for you to download the privacy policy as a PDF. The phone and fax number are included in the footer of that PDF. I have since added the PDF url to the `sources`.